wire-protoc-compatibility-tests/src/test/java/com/squareup/wire/Proto3WireProtocCompatibilityTests.kt

```diff
@@ -573,6 +574,35 @@ class Proto3WireProtocCompatibilityTests {
     assertThat(parsed).isEqualTo(value)
   }
 
+  @Test fun mapKeysAndValuesDefaultsToTheirRespectiveIdentityValue() {
+    // Build a MapType message with map_string_string field only, 2 entries, first one with key only, than value only.
+    // Both protoc and Wire encodes such with explicit zero-length fields, so had to be hand-crafted.
+    val bytes = listOf(
+      0x0a, // MapType.map_string_string tag -> 1|010 -> 10 -> 0a
+      0x04, // length
+      0x0a, // map key tag 1 -> 1|010 -> 10 -> 0a
+      0x02, // length
+      0x64, 0x65, // de
+      0x0a, // MapType.map_string_string tag -> 1|010 -> 10 -> 0a
+      0x04, // length
+      0x12, // map key tag 1 -> 1|010 -> 10 -> 0a
```

Comment:
Could you update this one? The comment is for the key, not the value. Same for the other test in proto2.

Reply:
Sure thing.

Reply:
Done.

Reply:
Thank you, I'll merge on green